src/waveform/renderers/allshader/waveformrendermark.cpp

```diff
@@ -398,9 +403,14 @@ void allshader::WaveformRenderMark::updatePlayPosMarkTexture() {
     imgwidth = 11.f;
     imgheight = height;
 
-    QImage image(static_cast<int>(imgwidth * devicePixelRatio),
-            static_cast<int>(imgheight * devicePixelRatio),
-            QImage::Format_ARGB32_Premultiplied);
+    const QSize size{static_cast<int>(std::lround(imgwidth * devicePixelRatio)),
+            static_cast<int>(std::lround(imgheight * devicePixelRatio))};
```

Comment:
Also here `makeRoundToPixel`? (same in the other renders)

Reply:
No, not the same.

Reply:
(Note that roundToPixel does both the division and a multiplication with the devicePixelRatio)

Reply:
ah right. Sorry.